NuRadioMC/SignalProp/analyticraytracing.py

```diff
@@ -1909,10 +1966,10 @@ def get_focusing(self, iS, dz, limit=2.):
             vetPos = copy.copy(self.__X1)
             recPos = copy.copy(self.__X2)
             recPos1 = np.array([self.__X2[0], self.__X2[1], self.__X2[2] + dz])
-        if(not hasattr(self, "_r1")):
-            self._r1 = ray_tracing(vetPos, recPos1, self.__medium, self.__attenuation_model, logging.WARNING,
-                             self.__n_frequencies_integration, self.__n_reflections)
-            self._r1.find_solutions()
+        self._r1 = ray_tracing(self.__medium, self.__attenuation_model, logging.WARNING,
```

Comment:
there was a reason for buffering results. I don't remember exactly what it was but I suspect that this slows down the code.

Reply:
Okay, I added the buffering back in